src/EchoToImageLUT.cpp

```diff
@@ -0,0 +1,123 @@
+#include "EchoToImageLUT.hpp"
+#include <algorithm>
+#include <iostream>
+
+using namespace radar_base;
+using namespace cv;
+using namespace std;
+
+int discretizeAngle(double theta_rad, int num_angles)
+{
+    double angle_step = 2 * M_PI / num_angles;
+    return round(theta_rad / angle_step);
+}
+
+EchoToImageLUT::EchoToImageLUT(int num_angles,
+    int sweep_size,
+    float beam_width,
+    int window_size)
+    : m_num_angles(num_angles)
+    , m_sweep_size(sweep_size)
+{
+    cout << "Setting Lookup table up\n";
+    setup(sweep_size, window_size, beam_width);
+}
+
+void EchoToImageLUT::setup(int sweep_size, int window_size, float beam_width)
+{
+    auto raw_data = computeRawLUTTable(sweep_size, window_size, m_num_angles, beam_width);
```

Comment:
`m_num_angles` is only used here, I think you could make it an argument of the `setup` method instead.

Reply:
I didn't touch this file, just imported from the temporary repo. I'll take a look into it next week.